main.tex

```diff
@@ -218,6 +221,8 @@
 
 \end{enumerate}
 
+\item Upon the passing of a Special Resolution by Lot Owners, a person, animal or agent (including an Owner or Occupier) may be denied the right of admission to an area of Common Property that is not needed for access to their Lot or its associated vehicle parking.
```

Comment:
I'm maintaining some ability to ban people, but this is now reliant on a special resulotion, not on a whim.